libs/coin-algorand/src/types.ts

```diff
@@ -20,6 +20,7 @@ export const AlgorandOperationTypeEnum = {
 export type AlgorandResources = {
   rewards: BigNumber;
   // This is the actual number of ASA opted-in for the Algo account
+  // test
```

Comment:
:eyes:

Reply:
yep it was to test that the CI runs in that case !